draft-ietf-dnsop-svcb-https.md

```diff
@@ -1190,7 +1193,7 @@ If this redirection would result in a loss of functionality (e.g. important
 resources that are only available on the "http" origin), the operator MUST
 NOT publish an HTTPS RR.
 
-When an HTTPS connection fails due to an error in the underlying secure
+When an "https" connection fails due to an error in the underlying secure
```

Comment:
```suggestion
When an HTTP connection fails due to an error in the underlying secure
```

Reply:
Given that HTTP connections don't generically have a secure transport, I find this version harder to understand.